.github/workflows/ci.yml

```diff
@@ -23,18 +23,6 @@ jobs:
       run: bundle exec rspec
     - name: Run rubocop
       run: bundle exec rubocop --display-cop-names
-# This doesn't work on PRs from forks due to GITHUB_TOKEN, which makes
-# it kinda useless for now
-# https://github.com/andrewmcodes/rubocop-linter-action/issues/68
```

Comment:
In theory, GH has a fix for this now, and using an inline action would be great for contributors...

Reply:
Indeed! I've got my eye on https://github.com/r7kamura/rubocop-problem-matchers-action, since it looks like the andrewmcodes repo is derelict. I've got a task for setting that up, though I want to handle that in separate PRs across the projects since this particular PR is to unblock some automation work

Reply:
LGTM